gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/ApiMetadata.java

```diff
@@ -18,16 +18,15 @@
 import io.gravitee.rest.api.model.MetadataFormat;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.With;
 
 @Builder
-@Data
```

Comment:
Why not ```@Getter``` & ```@Setter```?

Reply:
I mess-up I think I de-lombokize the class for a test but I forgot to revert back 🙈